platform/macos/CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Changelog for Mapbox Maps SDK for macOS
 
+* Fixed crash in `-[MGLStyle localizeLabelsIntoLocale:]` on iOS 9.3 (attempting to access a property that was introduced in iOS 10.0) ([#12123](https://github.com/mapbox/mapbox-gl-native/pull/12123))
```

Comment:
Similarly here

Reply:
For this cp I'm just adding leaving the related blurb. In the bump pr I will make this as it should be.